
package com.laegler.openbanking.soap.service;

import javax.xml.ws.WebFault;


/**
 * Not Acceptable
 *
 * This class was generated by Apache CXF 3.3.3
 * 2019-10-21T07:01:27.396+13:00
 * Generated source version: 3.3.3
 */

@WebFault(name = "PUT_Updatethebusinessorthetechnicalcontactsforthegivenorganisation_406", targetNamespace = "http://laegler.com/openbanking/soap/model")
public class PUTUpdatethebusinessorthetechnicalcontactsforthegivenorganisation406 extends Exception {

    private java.lang.Object putUpdatethebusinessorthetechnicalcontactsforthegivenorganisation406;

    public PUTUpdatethebusinessorthetechnicalcontactsforthegivenorganisation406() {
        super();
    }

    public PUTUpdatethebusinessorthetechnicalcontactsforthegivenorganisation406(String message) {
        super(message);
    }

    public PUTUpdatethebusinessorthetechnicalcontactsforthegivenorganisation406(String message, java.lang.Throwable cause) {
        super(message, cause);
    }

    public PUTUpdatethebusinessorthetechnicalcontactsforthegivenorganisation406(String message, java.lang.Object putUpdatethebusinessorthetechnicalcontactsforthegivenorganisation406) {
        super(message);
        this.putUpdatethebusinessorthetechnicalcontactsforthegivenorganisation406 = putUpdatethebusinessorthetechnicalcontactsforthegivenorganisation406;
    }

    public PUTUpdatethebusinessorthetechnicalcontactsforthegivenorganisation406(String message, java.lang.Object putUpdatethebusinessorthetechnicalcontactsforthegivenorganisation406, java.lang.Throwable cause) {
        super(message, cause);
        this.putUpdatethebusinessorthetechnicalcontactsforthegivenorganisation406 = putUpdatethebusinessorthetechnicalcontactsforthegivenorganisation406;
    }

    public java.lang.Object getFaultInfo() {
        return this.putUpdatethebusinessorthetechnicalcontactsforthegivenorganisation406;
    }
}
